src/app/submission/form/collection/submission-form-collection.component.html

```diff
@@ -25,7 +25,7 @@
             class="btn btn-outline-primary"
             (blur)="onClose()"
             (click)="onClose()"
-            [disabled]="(processingChange$ | async)"
+            [disabled]="(processingChange$ | async) || collectionModifiable"
```

Comment:
"collectionModifiable=true" should mean that the collection *can* be modified (i.e. dropdown should be enabled).  But, here it's the opposite... "collectionModifiable=true" means that this dropdown is **disabled**.

I'd recommend we change this to be `|| !collectionModifiable` and then change `workflowitems-edit-page-routing.module.ts` to set it to `false`.